include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -600,6 +600,287 @@ struct SPhysicalDeviceLimits
+        if (maxViewports > _rhs.maxViewports) return false;
+        if (maxViewportDims[0] > _rhs.maxViewportDims[0]) return false;
+        if (maxViewportDims[1] > _rhs.maxViewportDims[1]) return false;
+        if (viewportBoundsRange[0] > _rhs.viewportBoundsRange[0]) return false;
+        if (viewportBoundsRange[1] > _rhs.viewportBoundsRange[1]) return false;
+        if (viewportSubPixelBits > _rhs.viewportSubPixelBits) return false;
+        if (minMemoryMapAlignment > _rhs.minMemoryMapAlignment) return false;
+        if (bufferViewAlignment > _rhs.bufferViewAlignment) return false;
+        if (minUBOAlignment > _rhs.minUBOAlignment) return false;
+        if (minSSBOAlignment > _rhs.minSSBOAlignment) return false;
+        if (minTexelOffset > _rhs.minTexelOffset) return false;
+        if (maxTexelOffset > _rhs.maxTexelOffset) return false;
+        if (minTexelGatherOffset > _rhs.minTexelGatherOffset) return false;
+        if (maxTexelGatherOffset > _rhs.maxTexelGatherOffset) return false;
+        if (minInterpolationOffset > _rhs.minInterpolationOffset) return false;
+        if (maxInterpolationOffset > _rhs.maxInterpolationOffset) return false;
+        if (maxFramebufferWidth > _rhs.maxFramebufferWidth) return false;
+        if (maxFramebufferHeight > _rhs.maxFramebufferHeight) return false;
+        if (maxFramebufferLayers > _rhs.maxFramebufferLayers) return false;
+        if (maxColorAttachments > _rhs.maxColorAttachments) return false;
+        if ((framebufferColorSampleCounts.value & _rhs.framebufferColorSampleCounts.value) != framebufferColorSampleCounts.value) return false;
+        if ((framebufferDepthSampleCounts.value & _rhs.framebufferDepthSampleCounts.value) != framebufferDepthSampleCounts.value) return false;
+        if ((framebufferStencilSampleCounts.value & _rhs.framebufferStencilSampleCounts.value) != framebufferStencilSampleCounts.value) return false;
+        if ((framebufferNoAttachmentsSampleCounts.value & _rhs.framebufferNoAttachmentsSampleCounts.value) != framebufferNoAttachmentsSampleCounts.value) return false;
+        if ((sampledImageColorSampleCounts.value & _rhs.sampledImageColorSampleCounts.value) != sampledImageColorSampleCounts.value) return false;
+        if ((sampledImageIntegerSampleCounts.value & _rhs.sampledImageIntegerSampleCounts.value) != sampledImageIntegerSampleCounts.value) return false;
+        if ((sampledImageDepthSampleCounts.value & _rhs.sampledImageDepthSampleCounts.value) != sampledImageDepthSampleCounts.value) return false;
+        if ((sampledImageStencilSampleCounts.value & _rhs.sampledImageStencilSampleCounts.value) != sampledImageStencilSampleCounts.value) return false;
+        if ((storageImageSampleCounts.value & _rhs.storageImageSampleCounts.value) != storageImageSampleCounts.value) return false;
+        if (maxSampleMaskWords > _rhs.maxSampleMaskWords) return false;
+        if (timestampComputeAndGraphics > _rhs.timestampComputeAndGraphics) return false;
```

Comment:
Comparing booleans this way is interesting, may be correct but I think it was accidental :)))